config/locales/es.yml

```diff
@@ -62,3 +62,6 @@ es:
   errors:
     messages:
       not_in_district: El código postal no pertenece al distrito
+  census_authorization_handler:
+    not_old_enough: Menor de 16 años
+    not_in_census: No empadronado
```

Comment:
Why you add this translations?

Reply:
To fix tests that were failing in 0.26 and that were failing after update

Reply:
What tests?

Reply:
There were a few tests in 0.26 giving missing translation error. Upgrading to 0.27 haven't fixed them. Adding this translations fixed them.